### PR TITLE
Resolve composer memory limit issue, writable settings.php file

### PR DESCRIPTION
**Pull request by srowlands**
Added `COMPOSER_MEMORY_LIMIT` to prevent composer installation failures.
Ensure correct permissions on `settings.php`

### CODE REVIEW

File: .docker/Dockerfile.govcms8 [hunk 1]
```diff
@@ -13,6 +13,7 @@ RUN sed -i -e "/govcms\/govcms/ s!1.x!${GOVCMS_PROJECT_VERSION}!" /app/composer.
 
 COPY scripts/composer/ScriptHandler.php /app/scripts/composer/ScriptHandler.php
 
+ENV COMPOSER_MEMORY_LIMIT=-1
 RUN composer install -d /app
```
Comment (steveworley) [inline .docker/Dockerfile.govcms8]: Can we also add a composer cache clear here? 🤧

**Comment by tobybellwood** [comment]
On hold whilst we decide whether to pin drupal/core here or in govcms/govcms8 - instead of both :-)

File: composer.json [hunk 2]
```diff
@@ -22,12 +22,12 @@
         "cweagans/composer-patches": "^1.6.5",
         "drupal-composer/drupal-scaffold": "^2.5",
         "drupal/console": "^1.0.2",
-        "drupal/core": "8.6.x-dev",
+        "drupal/core": "8.7.1",
```
Comment (tobybellwood) [inline composer.json]: These were set to placeholder values to allow for the .env file to be definitive about a release - govcms version, drupal version, amazee version, php version
https://github.com/govCMS/govcms8lagoon/blob/master/.docker/Dockerfile.govcms8#L11-L12

but happy to discuss ;-)

Comment (srowlands) [reply composer.json]: Right! Totally missed when this change happened and clearly didn't RTFM ;)

Removed pinning, should still resolve composer memory install issue + writable settings.php file (warning in site reports).

**Pull request merged**